.github/workflows/jira_cloud_ci.yml

```diff
@@ -1,6 +1,11 @@
 name: cloud
 
 on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - "*"
```

Comment:
Isn't this the same as not specifying branches at all? Or do you plan to change it for `main` only?

```
on:
  push:
```

Reply:
i think I'll better do it on pull request only. New to the GHA world :-) Jenkins is my original thing. Thx